app/controllers/articles_controller.rb

```diff
@@ -72,6 +73,10 @@ def eds_init
       field.label = 'Abstract'
     end
 
+    config.add_search_field('series') do |field|
+      field.label = 'Series'
```

Comment:
@jkeck interesting! I just noticed @jvine's example search uses the field code `SE` and returns 5 results. My current query returns 59. So the search is supported, it's just not configured as a search field. Should I try to make a PR on the EBSCO repo?